documentation/src/main/asciidoc/topics/proc_propagation_hotrod_client_tracing.adoc

```diff
@@ -0,0 +1,21 @@
+[id='propagation-hotrod-client-tracing']
+= Hot Rod client tracing propagation
+The HotRod client fully supports the OpenTelemetry tracing propagation from the client to the Infinispan server.
+
+It means that if the OpenTelemetry tracing is configured both on client VM and on the {brandname} server,
```

Comment:
How do you configure tracing on client VM? Should we include the step?

Reply:
It depends on the client technology, not by us. For instance a Quarkus OpenTelemetry extension or a SpringBoot OpenTelemetry module can be added for instance.